core/templates/pages/topic-editor-page/topic-editor-page.component.ts

```diff
@@ -111,6 +111,19 @@ angular.module('oppia').directive('topicEditorPage', [
           ctrl.selectQuestionsTab = function() {
             TopicEditorRoutingService.navigateToQuestionsTab();
           };
+          ctrl.getNavbarText = function() {
+            if (TopicEditorStateService.hasLoadedTopic()) {
+              const activeTab = ctrl.getActiveTabName();
+              if (activeTab === 'main') {
+                return (
+                  `Topic Editor: ${ctrl.topic.getName()}
+                  (v${ctrl.topic.getVersion()})`);
```

Comment:
Just to check, do we need the newline and whitespace characters introduced because you are using a multiline a template string?

Reply:
The navbar screenshot is there, i think the white header text looks fine?